### DISCUSSION TITLE
Change order in startify theme

### DISCUSSION THREAD

**Discussion by vuki656** [goolord/alpha-nvim#78] [Q&A] [answered]
Hi,

Is it possible to change the order of groups in the startify theme? 

Currently, it's
1. MRU
2. MRU - Local

I'd like to swap those two so it's
1. MRU - local
2. MRU

**Comment by goolord** [comment, marked as answer]
yes,
```lua
local startify = require('alpha.themes.startify')
-- startify's `opts` as this `layout` field which lets you easily reorder sections
-- and the `section` table which has fields for the defaults for any sections you might want to use for a theme
startify.opts.layout = {
    { type = "padding", val = 1 },
    startify.section.header,
    { type = "padding", val = 2 },
    startify.section.top_buttons,
    startify.section.mru_cwd,
    startify.section.mru,
    { type = "padding", val = 1 },
    startify.section.bottom_buttons,
    startify.section.footer,
}
require'alpha'.setup(startify.opts)
```

what is *not* ideal at the moment is that you would need to also redefine `startify.section.mru_cwd` and `startify.section.mru` so that the buttons start at the correct number, which I didn't consider before. for the moment at least you can do that similarly to how i did above or how you redefine sections in the docs.

**Reply by vuki656** [reply]
It will do the trick. Thanks. 

The numbers are not a big deal.
